content/en/docs/reference/glossary/Quantity.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Quantity
+id: quantity
+date: 2023-01-24
+short_description: >
+ Quantity refers to a special notation which are portable across machines. 
+
+aka: 
+tags:
+- fundamental
+---
+
+Quantity describes how much of a resource, like CPU or memory, is allotted to a Pod where a container is running.
+For example, a pod may be allocated 1 CPU core and 2 gigabytes of memory, which would be specified as "1 CPU" and "2Gi" for the respective resources. These quantities are used by the [Kubernetes scheduler](https://kubernetes.io/docs/concepts/scheduling-eviction/kube-scheduler/) to determine which nodes have the available resources to run a pod, and to ensure that pods do not consume more resources than they are allocated.
```

Comment:
```suggestion
For example, a pod may be allocated 1 CPU core and 2 gigabytes of memory, which would be specified as "1 CPU" and "2Gi" for its respective resources. These quantities are used by the [Kubernetes scheduler](https://kubernetes.io/docs/concepts/scheduling-eviction/kube-scheduler/) to determine which nodes have the available resources to run a pod, and to ensure that pods do not consume more resources than they are allocated.
```
- Mention what resources are you talking about as well. 
- Also, I'm unsure of 'which quantities' are you mentioning about as a reader.
- You can take some references from [here](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/)
  as well.

Reply:
As said in first line ``resources like cpu and memory.``
I have added quantitites definition 
I dont understand how [this](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/) can be helpful. 
Regarding the text: you have suggested major changes. if I will try to add new things or update. It will be hard for beginners to get the idea. WDYT?

Reply:
sorry! Here's the correct [link](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) you can take some reference from.